pkg/registry/softwarecomposition/vulnerabilitymanifest/etcd.go

```diff
@@ -52,5 +52,5 @@ func NewREST(scheme *runtime.Scheme, storageImpl storage.Interface, optsGetter g
 	if err := store.CompleteWithOptions(options); err != nil {
 		return nil, err
 	}
-	return &registry.REST{store}, nil
+	return &registry.REST{Store: store}, nil
```

Comment:
yay golint!

Reply:
Kinda funny I found this with my LSP (`go vet`), since the linter did not even run on PR changes for me :)